.github/workflows/cron.yml

```diff
@@ -0,0 +1,62 @@
+name: crons
+
+on:
+  schedule:
+    - cron: "0 2 * * *"  # at 02:00 UTC
+
+jobs:
+  cron-gpu:
+    container:
+      image: nvcr.io/nvidia/pytorch:20.03-py3
+      options: "--gpus all"
+    runs-on: [self-hosted, linux, x64]
+    strategy:
+      matrix:
+        pytorch-version: [1.4.0, 1.5.1]
```

Comment:
Can we change 1.5.1 to the latest PyTorch? Then if PyTorch has a new version, we can automatically verify it?